lib/index.js

```diff
@@ -3756,13 +3762,21 @@ _.log = function () {
 /**
  * Wraps a node-style async function which accepts a callback, transforming
  * it to a function which accepts the same arguments minus the callback and
- * returns a Highland Stream instead. Only the first argument to the
- * callback (or an error) will be pushed onto the Stream.
+ * returns a Highland Stream instead.
+
+ * If no mapping hint is provided, only the first argument to the callback
```

Comment:
Like @svozza said, maybe move this to the Stream Objects section and put a link to that section from here?